api/SpeechSynthesisEvent.json

```diff
@@ -201,37 +201,53 @@
           "spec_url": "https://wicg.github.io/speech-api/#dom-speechsynthesisevent-elapsedtime",
           "support": {
             "chrome": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "chrome_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "edge": {
-              "version_added": "14"
-            },
-            "firefox": {
-              "version_added": "49"
-            },
+              "version_added": "14",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
+            },
+            "firefox": [
```

Comment:
This is _Option 2_ for recording the change - essentially create a new version and use notes in both to indicate the change. 

_Option 1_ [below](https://github.com/mdn/browser-compat-data/pull/13236/files#r740728356) is a single version with note. 

Option 3, not shown, would be some kind of subfeature.